### PR TITLE
Rewriting top CTAs and Earthly links

### PR DESCRIPTION
**Pull request by adamgordonbell**
Preview links:
https://6568bf28d174dd00789ecd2d--ephemeral-narwhal-3ac3b2.netlify.app/

Note: `topcta: false` marks blog post as not needing a CTA at the top. This is important because topcta.py is run weekly to add CTAs to new blog posts. I set it false where blog post is already about Earthly, or where a CTA at the top didn't seem to work with the content or what not needed.

### CODE REVIEW

File: blog/_posts/2021-01-26-monorepo-vs-polyrepo.md [hunk 1]
```diff
@@ -21,8 +21,7 @@ excerpt: |
     This article explores the debate between using a monorepo or a polyrepo structure for source code. It discusses the benefits and challenges of each approach, as well as the hybrid option. Whether you're interested in ease of importing, contributions within and across projects, viewing diffs, structure of releases, builds and CI, open-source considerations, or issue tracking, this article provides insights to help you make an informed decision.
 last_modified_at: 2023-07-11
 ---
-
-**We're [Earthly](https://earthly.dev/). We make building software simpler and therefore faster using containerization. This article talks about the benefits of using a Monorepo. Earthly is particularly useful if you're working with a Monorepo. [Check us out](/).**
+<!--sgpt-->**This article explores the advantages and disadvantages of monorepo and polyrepo strategies. Earthly excels at build automation for developers, regardless of whether they use a monorepo or a polyrepo. [Learn more](/).**
```
Comment (adamgordonbell) [inline blog/_posts/2021-01-26-monorepo-vs-polyrepo.md:24]: Another example.

File: blog/_posts/2021-01-15-top-3-resources-to-learn-golang-in-2021.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ excerpt: |
     Looking to learn GoLang? Check out this article for the top 3 resources recommended by an experienced developer. From interactive tours to in-depth explanations, these resources will help you get started and build familiarity with the language. Plus, there are bonus resources and recommendations for further learning. Don't miss out on this valuable guide!
 last_modified_at: 2023-08-28
 ---
-<!--sgpt-->**We're [Earthly](https://earthly.dev/). We make building software simpler and faster using containerization. If you're learning Golang, Earthly can be a useful tool for streamlining your build process. [Check us out](/).**
+<!--sgpt-->**In this article, you'll discover key resources for learning GoLang. Go developers looking to improve their build processes will find Earthly particularly useful. [Find out how Earthly can assist you](/).**
```
[the remark this answers is not in the document]
Comment (vladaionescu) [reply blog/_posts/2021-01-15-top-3-resources-to-learn-golang-in-2021.md:12]: Yeah, I think this is pretty good

File: blog/_posts/2022-01-12-stop-using-pie-charts.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ excerpt: |
     Learn why you should stop using pie charts and discover alternative plots that are easier to interpret. This article provides examples and code for creating horizontal bar charts, lollipop charts, and box plots using Python and Matplotlib.
 last_modified_at: 2023-07-19
 ---
-**We're [Earthly](https://earthly.dev/). We make building software simpler and faster with containerization. While you're exploring alternative plots, why not also explore a new way to build your Python projects? [Check it out](/).**
+****This article explores alternatives to traditional charts. Earthly significantly enhances DevOps teams' CI pipelines. [Check it out](https://cloud.earthly.dev/login).****
```
[the remark this answers is not in the document]
Comment (adamgordonbell) [reply blog/_posts/2022-01-12-stop-using-pie-charts.md]: Fixed

File: blog/_posts/2021-11-11-argocd-kubernetes.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ excerpt: |
     Learn how to simplify Kubernetes deployments using ArgoCD, a lightweight tool that reads environment configurations from a Git repository and applies changes to a Kubernetes cluster. Discover common use cases and step-by-step instructions for implementing ArgoCD in your project.
 last_modified_at: 2023-08-17
 ---
-**We're [Earthly](https://earthly.dev/). We make building software simpler and faster using containerization. If you're using ArgoCD for Kubernetes deployments, Earthly can enhance your workflow by automating and accelerating the build process. [Check it out](/).**
+**This article discusses the integration of ArgoCD in Kubernetes environments. Earthly enhances CI build processes deployments with reproducible builds. [Check it out](https://cloud.earthly.dev/login).**
```
[the remark this answers is not in the document]
Comment (adamgordonbell) [reply blog/_posts/2021-11-11-argocd-kubernetes.md:17]: It's probably possible but will take some doing. Let me think about it.

**Pull request merged**